sqldelight-compiler/src/test/table-interface-fixtures/requires-adapter/output/com/sample/Person.kt

```diff
@@ -22,6 +22,5 @@ public data class Person(
 ) {
   public class Adapter(
     public val friendsAdapter: ColumnAdapter<List<Person>, ByteArray>,
-    public val shhh_its_secretAdapter: ColumnAdapter<String, String>,
```

Comment:
This adapter isn't needed anymore, because we are now able to check the resolved types due to the fqn names.